boot/boot_serial/src/boot_serial.c

```diff
@@ -483,6 +504,11 @@ boot_serial_input(char *buf, int len)
         default:
             break;
         }
+    } else if (MCUBOOT_PERUSER_MGMT_GROUP_ENABLED == 1 &&
+               hdr->nh_group >= MGMT_GROUP_ID_PERUSER) {
```

Comment:
Worth taking https://github.com/zephyrproject-rtos/mcuboot/commit/bd69c3dd4b4b7cd0c8063c2d3a885c0664962bc7 in, that removes this line.